src/cli/onefuzz/job_templates/manage.py

```diff
@@ -7,10 +7,10 @@
 
 from onefuzztypes.job_templates import (
     JobTemplate,
-    JobTemplateCreate,
     JobTemplateDelete,
+    JobTemplateGet,
     JobTemplateIndex,
-    JobTemplateUpdate,
+    JobTemplateSave,
```

Comment:
WDYT about `Upload` instead of `Save`? To me, "save" sounds has the connotation of a local operation. We just want something concise that means "create-or-update", right?